src/org/traccar/geocoder/JsonGeocoder.java

```diff
@@ -65,7 +65,7 @@ private String handleResponse(
             return formattedAddress;
         } else {
             if (callback != null) {
-                callback.onFailure(new GeocoderException("Empty address"));
+                callback.onFailure(new GeocoderException("Empty address. " + json.getString("error_message")));
```

Comment:
I don't think this is correct. Are you sure every available geocoder use the same key?